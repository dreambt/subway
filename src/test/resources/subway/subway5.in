L5-1,L1-7