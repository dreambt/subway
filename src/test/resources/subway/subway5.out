6:L5-1,L5-2,L5-3,L5-4,X5,L5-5,L5-6,X4,X3,L1-5,L1-6,L1-7;6:L5-1,L5-2,L5-3,L5-4,X5,L5-5,L5-6,X4,L3-8,L3-9,X8,L1-7;6:L5-1,L5-2,L5-3,L5-4,X5,L1-11,X7,L1-10,L1-9,L1-8,X8,L1-7;6:L5-1,L5-2,L5-3,L5-4,X5,L1-11,X7,L4-6,L4-7,L4-8,X8,L1-7